pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: centrifuge_dart
+version: 0.0.1
+
+description: >
+  Dart client to communicate with Centrifugo from Flutter and VM over dart:io WebSocket
```

Comment:
Centrifuge and Centrifugo

Reply:
to work with Centrifugo there should be a separate method to optionally set connection JWT token, and to work with Centrifuge there should be a possibility to set headers so developer could pass custom authentication information to server